### PR TITLE
Update javascript snippets

### PR DESCRIPTION
**Pull request by wralith**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Update JS-SDK snippets, using v3.0.0

## Issue reference

#3280

### CODE REVIEW

**Review by shubham1172 — CHANGES_REQUESTED** [review]
Hi @wralith, thank you for this PR! The changes LGTM, but there seem to be some missing instances like 

1. https://github.com/dapr/docs/blob/ab7a3b5b25ac54d8adead5ee76b44fe8aa812b9c/daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md?plain=1#L276 
1. https://github.com/dapr/docs/blob/393e88fa13c00b78ef31bf1116a73ca828cc5b88/daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md?plain=1#L180

Can you please look at the missing ones and fix them too?

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ dapr run --app-id checkout --app-protocol http --dapr-http-port 3500 --resources
 In the `checkout` publisher service, we're publishing the orderId message to the Redis instance called `orderpubsub` [(as defined in the `pubsub.yaml` component)]({{< ref "#pubsubyaml-component-file" >}}) and topic `orders`. As soon as the service starts, it publishes in a loop:
 
 ```js
-const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT);
+const client = new DaprClient({ daprHost, daprPort });
```
Comment (shubham1172) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md]:
```suggestion
const client = new DaprClient();
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 2]
```diff
@@ -177,29 +177,19 @@ dapr run --app-id order-processor --resources-path ../../../resources/ -- npm ru
 The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
 
 ```js
-  const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT);
+const client = new DaprClient({ daprHost, daprPort, communicationProtocol })
```
Comment (shubham1172) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
const client = new DaprClient()
```

**Review by shubham1172 — COMMENTED** [review]
LGTM, small nits!

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md (hunk 1 above)
Comment (shubham1172) [reply daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md]: When we don't have any special values, we can omit these and fallback to default!

**Review by shubham1172 — APPROVED** [review]

**Comment by hhunter-ms** [comment]
@wralith can you pull in from v1.10? I've pushed a fix for the link validation error.

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**